source/elements/oneDPL/source/parallel_api.rst

```diff
@@ -5,7 +5,8 @@
 Parallel API
 ------------
 
-oneDPL provides the set of algorithms with execution policies as defined by the `C++ Standard`_.
+oneDPL provides the set of parallel algorithms as defined by the `C++ Standard`_,
+including parallel algorithms added in the 6th edition known as C++20.
```

Comment:
Do we want to say
```suggestion
including parallel algorithms added in the International Standard ISO/IEC 14882:2020 known as C++20.
```
?

Reply:
According to https://www.iso.org/standard/79358.html, C++20 is the 6th edition of the standard. Since a reference to the standard is already given in this sentence, I think it's quite clear what "the 6th edition" refers to.

I can do this change here, but then the other mentioning of the 6th edition below will look less clear.